app/controllers/ems_cloud_controller.rb

```diff
@@ -42,4 +42,61 @@ def restful?
 
   menu_section :clo
   has_custom_buttons
+
+  def sync_users
+    @ems = find_record_with_rbac(model, params[:id])
+    @in_a_form = true
+    drop_breadcrumb(:name => _("Sync Users"), :url => "/ems_cloud/sync_users")
+    @selected_admin_role = params[:admin_role]
+    @selected_member_role = params[:member_role]
+    @selected_password = params[:password]
+    @selected_verify = params[:verify]
```

Comment:
please, don't create instance variables, instead just use local variables and at the end of this method do an explicit `render` call and pass in the locals:

`render :template => ... , :locals => {:selected_admin_role => selected_admin_role..... }`

Reply:
Ok.